keyboards/handwired/uc2/config.h

```diff
@@ -0,0 +1,149 @@
+// Copyright 2022 Andy Chiu (@Andy Chiu)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/*
+ * Feature disable options
+ *  These options are also useful to firmware size reduction.
+ */
+
+/* disable debug print */
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x4143 // AC
+#define PRODUCT_ID   0x0002
+#define DEVICE_VER   0x0002
+#define MANUFACTURER AndyChiu
+#define PRODUCT      UniversalCentury2
+
+/* key matrix size */
+#define MATRIX_ROWS 12
+#define MATRIX_COLS 8
+
+#define MATRIX_ROW_PINS { D1,D0,D4,C6,D7,E6 }
+#define MATRIX_COL_PINS { F4,F5,F6,F7,B1,B3,B2,B6 }
+
+#define UNUSED_PINS
+
+#define DIODE_DIRECTION ROW2COL
+
+#define SOFT_SERIAL_PIN D2  // or D1, D2, D3, E6
+
+#define MASTER_RIGHT
+
+#define RGB_DI_PIN D3
+#define DRIVER_LED_TOTAL 108
+
+#ifdef RGB_DI_PIN
+#    ifdef RGBLIGHT_ENABLE
+#        define RGBLED_NUM 86 //108
+#        define RGBLIGHT_LED_MAP {71,65,59,53,49,50,54,60,66,72,76,80,85,81,77,73,67,61,55,56,62,68,74,78,82,83,84,79,75,69,63,57,51,47,46,48,52,58,64,70,40,39,33,27,21,8,5,0,1,2,3,4,15,11,14,13,10,7,6,9,12,16,17,18,19,20,22,23,24,25,26,28,29,30,31,32,34,35,36,37,38,42,41,43,44,45}
+#        define RGBLIGHT_HUE_STEP 8
+#        define RGBLIGHT_SAT_STEP 8
+#        define RGBLIGHT_VAL_STEP 8
+#        define RGBLIGHT_LIMIT_VAL 100 /* The maximum brightness level */
+#        define RGBLIGHT_SLEEP  /* If defined, the RGB lighting will be switched off when the host goes to sleep */
+/*== all animations enable ==*/
+#        define RGBLIGHT_ANIMATIONS
+/*== or choose animations ==*/
+//#    define RGBLIGHT_EFFECT_BREATHING
+//#    define RGBLIGHT_EFFECT_RAINBOW_MOOD
+//#    define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//#    define RGBLIGHT_EFFECT_SNAKE
+//#    define RGBLIGHT_EFFECT_KNIGHT
+//#    define RGBLIGHT_EFFECT_CHRISTMAS
+//#    define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//#    define RGBLIGHT_EFFECT_RGB_TEST
+//#    define RGBLIGHT_EFFECT_ALTERNATING
```

Comment:
Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```